packages/editor/src/core/types/node.ts

```diff
@@ -98,3 +98,10 @@ export type CellDrag = {
   type: 'cell';
   cell: PartialCell;
 };
+
+/**
+ * This is the ui-translator.
+ */
+export type UiTranslator = {
+  t: (label?: string) => string | undefined;
+};
```

Comment:
i would probably move it to the options.ts file instead of node.ts

Reply:
Will do